src/Compilers/CSharp/Test/Semantic/Semantics/AwaitExpressionTests.cs

```diff
@@ -119,6 +119,85 @@ async void Goo(Task<int> t)
             Assert.Equal("System.Boolean System.Runtime.CompilerServices.TaskAwaiter<System.Int32>.IsCompleted { get; }", info.IsCompletedProperty.ToTestDisplayString());
         }
 
+        [Fact]
+        [WorkItem("https://github.com/dotnet/roslyn/issues/76999")]
+        public void TestAwaitHoistedRef()
+        {
+            var src = """
+                using System.Threading.Tasks;
+
+                public sealed class Foo<T>
```

Comment:
>Foo [](http://example.com/codeflow?start=36&length=3)

Please do not use this word

Reply:
This is the submitted user code (see the issue)
But if you like I can modify the user code

Reply:
> what names should we use?

This word is flagged by policheck tool, Any word that cannot be interpreted as "offensive" is fine